src/QuadForm/Quad/ZLattices.jl

```diff
@@ -2908,21 +2973,21 @@ true
 function overlattice(glue_map::TorQuadModuleMap)
   S = relations(domain(glue_map))
   R = relations(codomain(glue_map))
+  @req ambient_space(S)==ambient_space(R) "lattices lie in different ambient spaces, try `primitive_extension` instead"
```

Comment:
```suggestion
  @req ambient_space(S) === ambient_space(R) "Lattices lie in different ambient spaces, try `primitive_extension` instead"
```
Here we want a strong equality I guess.. It should by default, but just to be sure